### PR TITLE
OIE Expression Language: New expressions for account mgmt policy

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds new OIE EL expressions related to account mgmt policy.
- **Is this PR related to a Monolith release?** Yes. 2024.07.2

### Resolves:

* [OKTA-746571](https://oktainc.atlassian.net/browse/OKTA-746571)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/b1c3a11519fa5cf7471448273b78250cd7c49e41/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d4e70e99-917c-42e1-b595-40143f50365d) | 86  | 65  | 78  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b9dacfc0-82e1-437e-ba31-bf535926f80a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/b5ecc9322f4cb24104ccd028df1cc195aedef103/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/485a987f-dea2-4da2-a099-35256401aff4) | 86  | 75  | 90  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4395576b-1ad0-4037-9229-c87d585e3f3d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/288bef223851b0c59127e10e1f3f6f84e9e63677/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cfbaa9f4-99ee-4ce9-b7ef-94be0d542738) | 86  | 75  | 90  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.550651da-886c-4c1e-a8db-3972b65ee541?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/ffaffe89790b5e0d4197605a2dd169a885609c46/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8dba9c1c-bc3a-4024-91d8-fe4242750571) | 86  | 75  | 90  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.6ac3613f-943f-4d10-8e71-3f78718887e6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/c205221fdf15ac6a908e2fb9feddb6ffefdbe086/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ace98c48-fb4e-4bb6-b41e-fe5b2a5d3596) | 86  | 72  | 90  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b2733b54-26d8-44e7-a972-1a42f84734cc?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md [hunk 1]
```diff
@@ -203,7 +214,7 @@ The `group.profile.name` key supports the operators `EXACT` and `STARTS_WITH` to
 
 ### Linked Object function
 
-Use this function to retrieve the User that is identified with the specified `primary` relationship. You can then access properties of that User.
+Use this function to retrieve the User that is identified with the specified `primary` relationship. You can then access the properties of that User.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md:217]:
```suggestion
Use this function to retrieve the user who's identified with the specified `primary` relationship. You can then access the properties of that user.
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md [hunk 2]
```diff
@@ -58,28 +61,36 @@ When you create an Okta expression, you can reference EDR attributes and any pro
...
+| `accessRequest.$operation`| `accessRequest` references the access context of the request. `operation` references the account management operation: `enroll`, `unenroll`, `recover`, or `unlockAccount`. | String |
+| `accessRequest.authenticator.$id` | `accessRequest` references the access context of the request. `authenticator.id` references an optional authenticator `id`. For example, the `id` of a custom authenticator. | String |
+| `accessRequest.authenticator.$key` | `accessRequest` references the access context of the request. `authenticator.key` references the [authenticator key](/docs/reference/api/policy/#authenticator-key-type-method-and-characteristic-relationships-for-constraints). | String |
 
 ## Functions
 
-Okta offers a variety of functions to manipulate properties to generate a desired output. You can combine and nest functions inside a single expression.
+Okta offers various functions to manipulate properties to generate a desired output. You can combine and nest functions inside a single expression.
 
 > **Note:** For the following expression examples, assume that the following properties exist in Okta and that the User has the associated values.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md:95]:
```suggestion
> **Note:** For the following expression examples, assume that the following properties exist in Okta and that the user has the associated values.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/d86375bdc39c9a739413e1cf7d48feb26501e29f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4e7d9d50-677c-4c45-86e8-3eead0d8ba12) | 86  | 72  | 90  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f28a2142-01bf-459f-95b6-35502e1ca4e7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/4d9e5163781319afe34fbf5ebf5f07fc24f211d0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/043ef3ac-d379-48ee-9955-3af45b2eb410) | 86  | 72  | 90  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3b310e64-952f-477e-a08c-579058944a83?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**